runtimerevolution/web3-roulette
# [WEB3-32] Encrypt user data

# Description

This PR updates the way user data is saved and fetched from the local storage. User information is encrypted before being saved, instead of being in plain text format. If the user tries to forge, the decryption will fail and the user will be redirected to the login page. A user context was created and the user info is fetched using Context API instead of being read several times from the local storage with the previous hook.

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-32

## Type of change

- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

1. Login in the app
2. Access local storage data and forge user data
3. Refresh the page or try to navigate to some page
4. Verify that you are redirected to the login page

## Checklist:
